File: remote/src/main/java/com/kelsos/mbrc/ui/views/MiniControlView.java
package com.kelsos.mbrc.ui.views;

import android.graphics.Bitmap;
import android.support.annotation.Nullable;

import com.kelsos.mbrc.annotations.PlayerState;

public interface MiniControlView {
  void updatePlayerState(@PlayerState.State String state);
  void updateTrack(String artist, String title);
  void updateCover(@Nullable Bitmap cover);
}

File: remote/src/main/kotlin/com/kelsos/mbrc/ui/views/MiniControlView.kt
package com.kelsos.mbrc.ui.views

import android.graphics.Bitmap

import com.kelsos.mbrc.annotations.PlayerState

interface MiniControlView {
  fun updatePlayerState(@PlayerState.State state: String)
  fun updateTrack(artist: String, title: String)
  fun updateCover(cover: Bitmap?)
}
